matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/DefaultTimeline.kt

```diff
@@ -349,6 +371,28 @@ internal class DefaultTimeline(
         }
     }
 
+    private fun getIdsOfPinnedEvents(): MutableList<String> {
+        return stateEventDataSource
+                .getStateEvent(roomId, EventType.STATE_ROOM_PINNED_EVENT, QueryStringValue.Equals(""))
+                ?.getIdsOfPinnedEvents() ?: mutableListOf("")
```

Comment:
you can replace `?: mutableListOf("")` by `orEmpty()`, which is also more correct, since I do not see the point of having `""` in the list.